pkg/util/config/config.go

```diff
@@ -108,6 +108,7 @@ type Config struct {
 	ConnectionPool
 
 	WatchedNamespace      string            `name:"watched_namespace"`    // special values: "*" means 'watch all namespaces', the empty string "" means 'watch a namespace where operator is deployed to'
+	KubernetesUseConfigMaps                bool              `name:"kubernetes_use_configmaps" default:"false"`
```

Comment:
please `go.fmt` all your code. Usually, your IDE should auto-align here.

Reply:
oh, right, not sure why it didn't.
Did it now for all of them.